subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/LazyDownloadsIntegrationTest.groovy

```diff
@@ -67,16 +66,16 @@ class LazyDownloadsIntegrationTest extends AbstractHttpDependencyResolutionTest
         succeeds("graph")
     }
 
-    @ToBeFixedForConfigurationCache(because = "Uses Configuration API in task action")
-    def "downloads only the metadata when resolved artifacts are queried"() {
+    def "downloads only the metadata when resolved dependencies are queried"() {
```

Comment:
This test is now identical to the test above. 

Is this behavior getting tested elsewhere? I feel like we are removing lot of test coverage for `ResolvedConfiguration` in this PR even though we are leaving it un-deprecated and will need to continue supporting it.

Reply:
Good catch, Sterling's update addressed this.

Reply:
Meaning we can delete this one now?

Reply:
This one is slightly different since it looks at the metadata from the resolution result, but it's probably not that interesting to check separately.

Reply:
It prints it out but then seemingly does nothing with it. Am I missing something?

Reply:
It is just printing it out to force retrieval from the repo, so the `when` block can verify the requests.  Without the `println`, no requests, test fails.

Reply:
The actual `println` isn't doing the retrieval though, right? It's the call to `get()`. I am confused kn how the println itself is doing that

Reply:
It is just the `get`

Reply:
I was working with the `println` line as a whole, yes.